alphagov/paas-cf-conduit
# Replace go-bindata with inline code

## What

The `application.zip` file that was committed and bundled with the code
using the external library go-bindata contains a single `Staticfile` with no
content.

We can reproduce the same content by using the `archive/zip` standard
library which requires a less complicated build process and prevents a
slightly confusing error if you try to `go get` or `go build` the repo
without using the `Makefile`.

I wasn't technically reviewing this story but I had to look into something and then I had this itch to scratch.

## How to review

- code review
- checkout this branch
- run `make install`
- try running `cf conduit <service_instance>` against a CF environment and confirm that it works
